packages/components/src/utils/validation/validatePropChange.ts

```diff
@@ -0,0 +1,4 @@
+import { deepEqual } from '../jss';
+
+export const validatePropChange = (newVal, oldVal, propertyName, arrayOfRelevantPropNames): boolean =>
```

Comment:
Does't this produce tons of lint warnings since typings are missing?
Also the functions name suggests it is validating something and throws an error.